types/reveal/index.d.ts

```diff
@@ -1,241 +1,1701 @@
+         */
+        interface Options {
+            // Presentaion size
+            // https://github.com/reveal/revealjs.com/blob/master/src/presentation-size.md
+
+            /**
+             * The "normal" size of the presentation, aspect ratio will be preserved
+             * when the presentation is scaled to fit different resolutions
+             *
+             * @defaultValue `960`
+             */
+            width?: number;
+
+            /**
+             * The "normal" size of the presentation, aspect ratio will be preserved
+             * when the presentation is scaled to fit different resolutions
+             *
+             * @defaultValue `700`
+             */
+            height?: number;
+
+            /**
+             * Factor of the display size that should remain empty around the content
+             *
+             * @defaultValue `0.04`
+             */
+            margin?: number;
+
+            /**
+             * Bounds for smallest/largest possible scale to apply to content
+             *
+             * @defaultValue `0.2`
+             */
+            minScale?: number;
+
+            /**
+             * Bounds for smallest/largest possible scale to apply to content
+             *
+             * @defaultValue `2.0`
+             */
+            maxScale?: number;
+
+            /**
+             * Display presentation control arrows
+             *
+             * @defaultValue `true`
+             */
+            controls?: boolean;
+
+            /**
+             * Help the user learn the controls by providing hints, for example by
+             * bouncing the down arrow when they first encounter a vertical slide
+             *
+             * @defaultValue `true`
+             */
+            controlsTutorial?: boolean;
+
+            /**
+             * Determines where controls appear, "edges" or "bottom-right"
+             *
+             * @defaultValue `bottom-right`
+             */
+            controlsLayout?: 'edges' | 'bottom-right';
+
+            /**
+             * Visibility rule for backwards navigation arrows; "faded", "hidden"
+             * or "visible"
+             *
+             * @defaultValue `faded`
+             */
+            controlsBackArrows?: 'faded' | 'hidden' | 'visible';
+
+            /**
+             * Display a presentation progress bar
+             *
+             * @defaultValue `true`
+             */
+            progress?: boolean;
+
+            /**
+             * Display the page number of the current slide
+             * - true: Show slide number
+             * - false: Hide slide number
+             * Can optionally be set as a string that specifies the number formatting:
+             * - "h.v": Horizontal . vertical slide number (default)
+             * - "h/v": Horizontal / vertical slide number
+             * - "c": Flattened slide number
+             * - "c/t":	Flattened slide number / total slides
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/reveal/index.d.ts;
  Line: 949;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.